frontend/src/stores/product.js

```diff
@@ -43,6 +43,15 @@ export const useProductStore = defineStore('productStore', {
 
       return uniqueProducts;
     },
+    /**
+     * Get all trending products.
+     * 
+     * @param {object} state - The state object.
+     * @returns {array} - The list of products that are trending.
+     */
+    trendingProducts: (state) => {
+      return state.products.filter(product => product.trending_now);
```

Comment:
What is `trending_now`?, and where is it defined in backend site?

Reply:
You can see the trending_now in Product model.